### DISCUSSION TITLE
Dynamic Linked Forms

### DISCUSSION THREAD

**Discussion by arjunarasu** [Budibase/budibase#2547] [General]
Hi,

Is there any way to create Budibase Forms via something like react json forms?

ie: https://rjsf-team.github.io/react-jsonschema-form/

**Comment by shogunpurple** [comment]
Hi @arjunarasu - this is not possible with budibase. We have plans in the future to expose a complete REST API for budibase, which might allow you to set something like this up with your own custom front end.

**Comment by arjunarasu** [comment]
thank you, its a fantastic solution btw.
